src/components/Catalyst/Step4.js

```diff
@@ -183,6 +183,7 @@ const Step4 = ({
     if (isEasyConfirmationEnabled) {
       onContinue()
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think the rule is right here, `isEasyConfirmationEnabled` should be in the dep list, don't you think?

Reply:
this change has some cascading effects
ill follow the rules, but it's gonna need some manual verification